code/game/mecha/equipment/weapons/melee.dm

```diff
@@ -24,6 +34,43 @@
 	var/icon/melee_overlay
 	force = 60 // Big sword make big boo boo - R4d6 / Upped further because delay - Wizard
 
+/obj/item/mecha_parts/mecha_equipment/melee_weapon/sword/attach(obj/mecha/M as obj)
```

Comment:
This will use the default mech-sword overlay, should probably make a proper one for the cutlass and update the code to reflect.

Reply:
I will most likely sit down over the weekend and make a proper sprite for it. However at the current moment in time, it wouldn't be a issue pushing it without the overlay. Most weapons are invisible upon mechs anyway. Though it is something I intend on doing soon.